From 4201c63a110c1b7a5212dd3030065e8ad52b5e90 Mon Sep 17 00:00:00 2001
From: Adrian DC <radian.dc@gmail.com>
Date: Wed, 9 May 2018 01:27:51 +0200
Subject: [PATCH 10/12] backuptool: Temporarily render version check permissive

 * Due to both following commits, backuptool went permissive
    and lineage properties got lost from the system on devices
    that do not have BOARD_PROPERTY_OVERRIDES_SPLIT_ENABLED

    "backuptool: Take into account new location for system default props"
    Change-Id: I62046447876c2198a0c4f88a4f36f4723d417617

    "lineage: Move to Google's method of defining system default props"
    Change-Id: I6cb0e28a7599b010b389cc541015a37010a00f4b

 * Once BOARD_PROPERTY_OVERRIDES_SPLIT_ENABLED is globally enabled,
    a period of time is required for "most" of the users to upgrade
    their system with fixed lineage properties before breaking addons
    by repairing the backuptool script

Change-Id: Iea8865ea9bb05eed56a8a0a7b95e3f04b01c4bae
---
 prebuilt/common/bin/backuptool.sh | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/prebuilt/common/bin/backuptool.sh b/prebuilt/common/bin/backuptool.sh
index 4e3385e3..8d85a643 100755
--- a/prebuilt/common/bin/backuptool.sh
+++ b/prebuilt/common/bin/backuptool.sh
@@ -48,10 +48,10 @@ check_prereq() {
 if [ ! -r /system/build.prop ]; then
     return 0
 fi
-if ! grep -q "^ro.lineage.version=$V.*" /system/etc/prop.default /system/build.prop; then
-  echo "Not backing up files from incompatible version: $V"
-  return 0
-fi
+# if [ ! grep -q "^ro.lineage.version=$V.*" /system/etc/prop.default /system/build.prop ]; then
+#   echo "Not backing up files from incompatible version: $V"
+#   return 0
+# fi
 return 1
 }
 
-- 
2.17.0

